smartsim/_core/_cli/build.py

```diff
@@ -419,16 +445,29 @@ def check_backends_install(self) -> None:
         if os.environ.get("RAI_PATH"):
             if installed_redisai_backends():
                 logger.error(
-                    f"There is no need to build. Backends are already built and specified in the environment at 'RAI_PATH': {CONFIG.redisai}"
+                    (
+                        "There is no need to build. Backends are already built and "
+                        "specified in the environment at 'RAI_PATH': {}"
+                    ),
+                    CONFIG.redisai,
                 )
             else:
                 logger.error(
-                    f"Before running 'smart build', unset your RAI_PATH environment variable with 'unset RAI_PATH'."
+                    (
+                        "Before running 'smart build', unset your RAI_PATH environment "
+                        "variable with 'unset RAI_PATH'."
+                    )
```

Comment:
parens unneeded?

Reply:
without them, the string breaks line length limit. a bunch had to be broken for this reason. E.g. 

```
************* Module smartsim._core._cli.build
smartsim/_core/_cli/build.py:456:0: C0301: Line too long (115/88) (line-too-long)
```